File: src/main/java/com/trabalho_testes/trabalho_testes/entidade/AcaoJogador.java
package com.trabalho_testes.trabalho_testes.entidade;

import lombok.Data;

import javax.persistence.*;

@Data
@Entity
public class AcaoJogador {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private Integer total;

    @ManyToOne
    private Jogador jogador;

    @ManyToOne
    private Acao acao;

    @ManyToOne
    private Jogo jogo;
}

File: frontend/src/app/app/servico/acao-jogador/acao-jogador.ts
import { IJogador, Jogador } from '../jogador/jogador';

export class IAcaoJogador {
    id: number;
    total: number;
    jogador: IJogador;
}

export class AcaoJogador {
    id: number;
    total: number;
    jogador: Jogador;
}